documentation/mkdocs.yml

```diff
@@ -7,6 +7,7 @@ nav:
         - 'Overview': infrastructure/overview.md
         - 'Custom Jenkins Setup': infrastructure/customjenkins.md
         - 'Vault For Pipline Secrets': infrastructure/vault.md
+        - 'Fixing docker rate limit': infrastructure/docker-rate-limit.md
```

Comment:
Not sure if I would call it "fixing" ...

Reply:
Suggestions welcome. I had "troubleshooting" before, but that word is too long and makes the navigation bar look bad..